examples/arNFT_gltf_brave_robot_example.html

```diff
@@ -21,22 +21,24 @@
     <script>
 			ARnft.ARnft.init(640, 480, "examples/DataNFT/pinball", 'config_brave_robot.json', true, true)
 			.then((nft) => {
-				document.addEventListener('getThreejsRenderer', (ev) => {			
-				    let renderer = ev.detail.renderer.renderer
-				    renderer.outputEncoding = THREE.sRGBEncoding;
-					renderer.physicallyCorrectLights = true;
-					console.log('setting texture outputEncoding and physicallyCorrectLights');
-				})
 				document.addEventListener('onInitThreejsRendering', (ev) => {
 					console.log(ev.detail);
 					const renderer = ev.detail.renderer;
 					console.log(renderer);
 					const scene = ev.detail.scene;
 					console.log(scene);
+					const camera = ev.detail.camera;
+					console.log(camera);
 					let directionalLight = new THREE.DirectionalLight('#fff', 0.4);
 					directionalLight.position.set(0.5, 0, 0.866);
 					scene.add(directionalLight)
 				})
+				document.addEventListener('onAfterInitThreejsRendering', (ev) => {			
```

Comment:
i think that to set outputEncoding and lights would be better to set before this, This is only to show the difference beween the two.

Reply:
i mean that would be correct set them in `onInitThreejsRendering`, maybe this can be confusing. I will move them and i add some comments.